content/en/docs/v0.3.0/Examples/AnnotatedArbitraryGenerator.md

```diff
@@ -4,7 +4,7 @@ linkTitle: "AnnotatedArbitraryGenerator"
 weight: 4
 ---
 {{< alert color="secondary" title="Note">}}
-Predefined AnnotatedArbitraryGenerator can be found [here]({{< relref "/docs/features/defaultsupportedtypes" >}})
+Predefined AnnotatedArbitraryGenerator can be found [here]({{< relref "/docs/v0.3.0/features/defaultsupportedtypes" >}})
```

Comment:
저희가 마이너 버전 단위로 관리할거면 0.3.x, 0.4.x 또는 0.3, 0.4 로 하면 어떨까 싶습니다.

Reply:
0.3.0, 0.3.1 와 같이 세부적으로 나누려고 했는데, breaking change 되는 버전 위주로 하는 게 좋겠군요
반영하겠습니다.

Reply:
@seongahjo  넵 patch 버전마다 복사해서 분리해서 만들면 너무 많아질거 같아서요
0.3.1 0.3.2 등의 패치버전의 변경은 Release Notes 에 작성해놓고
Documents 안에 내용 추가할 때 Since 0.3.1 이런식으로 명시해놓으면 구분이 될거 같습니다.